vector/src/main/java/im/vector/app/features/home/HomeActivity.kt

```diff
@@ -274,7 +277,7 @@ class HomeActivity :
         intent?.dataString?.let { deepLink ->
             val resolvedLink = when {
                 // Element custom scheme is not handled by the sdk, convert it to matrix.to link for compatibility
-                deepLink.startsWith(MATRIX_TO_CUSTOM_SCHEME_URL_BASE) -> {
+                deepLink.startsWith(MATRIX_TO_CUSTOM_SCHEME_URL_BASE)    -> {
```

Comment:
Rather not touch indention here: we care more about having less merge conflicts with upstream, then proper code style